recipes/basictools/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.1" %}
+
+package:
+  name: basictools
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/DRTI/basic-tools/-/archive/v{{ version }}/basic-tools-v{{ version }}.tar.bz2
+  sha256: 7e4354044714c5a5d815e000109940fbb7dd4edce3140a8b51bcc04ee720c333
+  patches:
+    # disable mkl, openmp
+    - setup.patch
+
+build:
+  number: 0
+  script: "CC=${CXX} EIGEN_INC=${PREFIX}/include/eigen3 ${PYTHON} -m pip install --no-deps . -vv"  # [unix]
+  script: "set EIGEN_INC=%LIBRARY_INC%\\eigen3 & %PYTHON% -m pip install --no-deps . -vv"  # [win]
+  skip: true  # [py2k or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
+    - eigen
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - scikit-sparse  # [not win]
+    - pyamg
+    - sympy
+    - h5py
+    - pyparsing
+    - mpi4py  # [not win]
+    - vtk
+    - matplotlib
```

Comment:
Can this be `matplotlib-base` so we don't pull in a Qt dependency?

Reply:
thanks, it'll try it